docs/ref/dotz.md

```diff
@@ -417,6 +417,7 @@ Values for V3.5+ are shown below in bold type.
 os               | 32-bit  | 64-bit
 -----------------|---------|--------
 Linux            | **l32** | **l64**
+Linux on Arm     |         | l64 (reports l64arm since 4.1t 2022.09.02)
```

Comment:
1. Should these be in bold type? (Values for V3.5+ are in bold according to the prev line.)
2. Should the "arm" in "164arm" also be capitals?
3. See suggestion:

```suggestion
Linux on ARM     |         | l64 (reports l64arm since 4.1t 2022.09.02)
```

Reply:
> Should the "arm" in "164arm" also be capitals?
no - that what the system outputs